docs/layers/introduction.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Introduction
+sidebar_label: Introduction
+slug: /layers
+---
+
+# What are Layers?
```

Comment:
Layers already existed as a way to isolate experiments from eachother.  So we definitely need to call that out as well, as that functionality remains (and is another major reason to use them)